README.md

```diff
@@ -27,6 +27,14 @@ To dump data to seed-data dump file (i.e. for use by other devs):
 pg_dump -U_postgres mylibrarynyc -f db/seed-data.sql --data-only --exclude-table="schema_migrations|holds|users" --inserts --column-inserts
 ```
 
+To Import New NYC Schools:
+
+Assuming your CSV is public data, upload it under root/data/public.
+
+Run this command from the console locally and on AWS (replace the filename in the command): `rake ingest:import_all_nyc_schools['data/public/2016_-_2017_School_Locations.csv']`
+
+If the school is not found by zcode, the rake task will create a new record.  If the school is found by zcode, it will overwrite school's name, address_line_1, state, address_line_2, borough, postal_code, and phone_number with the data in the CSV.
```

Comment:
@nnat425 does this documentation look sufficient?  This PR is ready for another look.  Gracias!